src/App.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import InputTodo from './InputTodo';
+import TodoList from './TodoList';
```

Comment:
```suggestion
import React, { useState } from 'react';

import InputTodo from './InputTodo';
import TodoList from './TodoList';
```

의미상으로 구분되는 경우 빈 줄을 넣어서 코드를 구분해주세요.
코드 빈 줄을 하나에도 의미가 담겨있다고 생각하시면 도움이 되실거에요.